paper/src/main/java/com/denizenscript/denizen/paper/properties/PaperWorldExtensions.java

```diff
@@ -120,20 +85,20 @@ public void adjust(Mechanism mechanism) {
         // @tags
         // <WorldTag.no_tick_view_distance>
         // -->
-        if (mechanism.matches("no_tick_view_distance")) {
+        WorldTag.tagProcessor.registerMechanism("no_tick_view_distance", false, ElementTag.class, (object, mechanism, input) -> {
             BukkitImplDeprecations.paperNoTickViewDistance.warn(mechanism.context);
-            if (!mechanism.hasValue()) {
-                world.getWorld().setNoTickViewDistance(-1);
+            if (!input.isInt()) {
+                object.getWorld().setNoTickViewDistance(-1);
             }

```

Comment:
Looks like this allows no input? should use the normal `registerMechanism` method instead of the one that requires input